system/netdata.logrotate.in

```diff
@@ -0,0 +1,13 @@
+@localstatedir_POST@/log/netdata/*.log {
+	daily
+	missingok
+	rotate 14
+	compress
+	delaycompress
+	notifempty
+	sharedscripts
+	postrotate
+		systemctl restart netdata.service
```

Comment:
oh! sorry, I did not notice that before.
do we must use systemd specific commands? and restart?
worse case logrotate should use the mode of copytruncate.